modules/oe_content_featured_media_field/src/Plugin/Field/FieldWidget/FeaturedMediaEntityBrowserWidget.php

```diff
@@ -262,6 +276,25 @@ protected function formElementItemsEntities(FieldItemListInterface $items, int $
     $entity_type = $this->fieldDefinition->getFieldStorageDefinition()->getSetting('target_type');
     $entity_storage = $this->entityTypeManager->getStorage($entity_type);
 
+    $element_path = array_merge($element['#field_parents'], [$this->fieldDefinition->getName(), $delta]);
+    $input_exists = NULL;
+    $input_value = NestedArray::getValue($form_state->getUserInput(), $element_path, $input_exists);
+    $value_exists = NestedArray::keyExists($form_state->getValues(), $element_path);
+
+    if (!$value_exists && $input_exists && !is_array($input_value['target_id'])) {
+      $data = empty($input_value['target_id']) ? [] : explode(' ', trim($input_value['target_id']));
+      if ($data) {
+        $data = reset($data);
+        $value['target_id'] = explode(':', $data)[1];
+        $items->get($delta)->setValue($value);
+        $entity = $entity_storage->load($items[$delta]->target_id);
+        if (!empty($entity)) {
+          $entities[] = $entity;
+        }
+        return $entities;
+      }
+    }
```

Comment:
The original widget has a submit callback that sets into the form the value and then rebuild the form. That should be enough to have the values correctly populated. I did delete all this code, copied from the parent widget:
```
'#submit' => [[get_class($this), 'entityBrowserValueUpdated']],
'#limit_validation_errors' => [array_merge($element['#field_parents'], [$this->fieldDefinition->getName()])],
'#executes_submit_callback' => TRUE,
``` 
Then you need to override the method as we handle one value per field only. This way we get completely rid of user input mingling. Tests are green so I assume it's ok.
Also [fago says](https://www.drupal.org/project/entity_browser/issues/2851580#comment-13732789) that the above code seems not useful

Reply:
True. However, it seems I still need to check the input for the case in which an element is removed (and prevent defaults from $items). Could not find a better way. But indeed, using the #submit, we dont need to anymore load values here,

Reply:
True, it works for the default widget cause they have the same code around.